### PR TITLE
RHDEVDOCS-5996: Content creation for the GitOps 1.10.4 RN

### PR DESCRIPTION
**Pull request by Dhruv-Soni11**
Version(s):

1.10.4

Content from this release must be CP'ed to GitOps 1.12, 1.11, and 1.10 releases.

Issue:

https://issues.redhat.com//browse/RHDEVDOCS-5996

Link to docs preview:

https://74136--ocpdocs-pr.netlify.app/openshift-gitops/latest/release_notes/gitops-release-notes.html#release-notes-for-gitops-1-10-4_gitops-release-notes

SME review: @reginapizza
QE review: @varshab1210
Internal peer review: @eromanova97
Peer review:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by Dhruv-Soni11** [comment]
Kindly note that the content for this release has been populated as per the inputs provided in this [doc](https://docs.google.com/document/d/1U1N8fwbYrwrCv4MENsU3ReD8HWjAEBeP1EGsPQuso_k/edit). The JIRA tracker links are populated for each Fixed Issue and the content for the embargo issue is not added yet.

File: modules/gitops-release-notes-1-10-4.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+{gitops-title} 1.10.4 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-10-4_{context}"]
+== Errata updates
+
+[id="rhsa-XXXX-NNNN-gitops-1-10-4-security-update-advisory_{context}"]
+=== RHSA-XXXX-NNNN - {gitops-title} 1.10.4 security update advisory
+
+Issued: 2024-04-03
```
[the remark this answers is not in the document]
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-10-4.adoc:18]: Fixed.

**Review by reginaelyse — APPROVED** [review]

**Comment by varshab1210** [comment]
LGTM

**Comment by Dhruv-Soni11** [comment]
> /remove-label peer-review-needed /remove-label peer-review-in-progress /label peer-review-done
> 
> LGTM
> 
> FYI - looks like you will need to rebase

Hi @obrown1205 - I just did a quick rebase and it looks fine. Thanks.

**Review by adellape — APPROVED** [review]

**Pull request merged**